sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece-response.processing.service.ts

```diff
@@ -74,18 +70,23 @@ export class ECEResponseProcessingService {
    * @returns Process summary result.
    */
   async process(): Promise<ProcessSummaryResult[]> {
-    // Get all the institution codes who are enabled for integration.
-    const integrationEnabledInstitutions =
-      await this.institutionLocationService.getAllIntegrationEnabledInstitutionCodes();
-    const filePathDetails: InstitutionFileDetail[] =
-      integrationEnabledInstitutions.map((institutionCode) => ({
-        path: path.join(
-          this.institutionIntegrationConfig.ftpResponseFolder,
-          institutionCode,
-          ECE_RESPONSE_FILE_NAME,
-        ),
-        institutionCode,
-      }));
+    // Get all the ECE response files from the SFTP location.
```

Comment:
In the ECE integration service `checkIfFileExist` optional parameter is no more required. If we remove that we will reduce the overhead to check file existence in the implementation. 

![image](https://github.com/user-attachments/assets/e617d737-0a86-48c5-b99a-2502e5fbc9a6)

![image](https://github.com/user-attachments/assets/6f3c4887-e805-4562-a692-7cc94188bba9)